playbooks/openstack-deploy_network.yml

```diff
@@ -0,0 +1,40 @@
+---
+- hosts: localhost
+  gather_facts: no
+  connection: local
+  post_tasks:
+    - name: Load generic vars
+      include_vars: "{{ playbook_dir }}/../vars/common-vars.yml"
+    - name: Load openstack vars
+      include_vars: "{{ playbook_dir }}/../vars/deploy-on-openstack.yml"
+
+    - name: Check if the network stack already exist
+      command: openstack stack show "{{ deploy_on_openstack_network_stackname }}" -c id -f value
+      # rc=1 means stack does not exist, rc=0 means stack does exist
+      ignore_errors: yes
```

Comment:
you probably want to remove ignore_errors: yes

and replace it by:
`failed_when:  network_stack_available.rc != 0 or network_stack_available.rc != 1` or `failed_when: False`

You probably want to add `changed_when: False` too, because it's a read only operation, you'll never change things here.

However, I believe that the os_stack is idempotent, so you probably don't need to do all of this.

Reply:
Ah. Thanks for the hints. I thought `os_stack` would create a new stack whenever you call it. But it is updating the stack when the stack is already available. Great!

Reply:
Fixed